string2.py

```diff
@@ -31,8 +35,11 @@ def verbing(s):
 # So 'This dinner is not that bad!' yields:
 # This dinner is good!
 def not_bad(s):
-    """Your code goes here.  Edit this docstring."""
-    return
+  if s.find("bad") > s.find("not"):
```

Comment:
Try to do the .find operations just once for "not" and "bad"; this will save time.  Each time you do a .find, it has to run through all the characters in the string again as it looks for the substring.